pkg/unikontainers/config_test.go

```diff
@@ -78,12 +78,21 @@ func TestGetConfigFromSpec(t *testing.T) {
 		}
 
 		expectedConfig := &UnikernelConfig{
-			UnikernelType: "type1",
+			UnikernelType:    "type1",
+			UnikernelVersion: "",
```

Comment:
Let;s not define empty fields for this test. Except if it might trigger any further issue.